tests/qadence/test_mitigation.py

```diff
@@ -245,6 +244,51 @@ def test_analog_zne_with_pulser(
     options = {"noise_probas": noise_probas}
     noise = Noise(protocol=noise_type, options=options)
     mitigation = Mitigations(protocol=Mitigations.ANALOG_ZNE)
-    noisy_expectation = model.expectation(noise=noise, mitigation=mitigation)
     exact_expectation = model.expectation()
-    assert torch.allclose(noisy_expectation, exact_expectation, atol=1.0e-2)
+    mitigated_expectation = model.expectation(noise=noise, mitigation=mitigation)
+    assert torch.allclose(mitigated_expectation, exact_expectation, atol=1.0e-2)
+
+
+@pytest.mark.parametrize(
+    "analog_block, observable, noise_probas, noise_type, param_values",
+    [
+        (
+            chain(AnalogRX(pi / 2.0), AnalogRZ(pi)),
+            [Z(0) + Z(1)],
+            torch.tensor([0.1]),
+            Noise.DEPOLARIZING,
+            {},
+        ),
+        # (
+        #     # Parameter time and harcoded angle for Bell state preparation.
+        #     chain(
+        #         entangle("t", qubit_support=(0, 1)),
```

Comment:
rm

Reply:
Not yet. I need some input from @jpmoutinho to uncomment this. Will add a fixme about it.